### DISCUSSION TITLE
Add Helm values to support insecure registries

### DISCUSSION THREAD

**Discussion by ecniiv** [aquasecurity/starboard#1000] [Development]
Hi,

In the starboard-trivy-configmap, there is no possibility to allow trivy insecure connections to a container registry with SSL activated. So, in my helm chart I added this in the configmap: 

```
{{- range $key, $registry := .insecureRegistries }}
  trivy.insecureRegistry.{{ $key }}: {{ $registry | quote }}
{{- end }}
```
and in the values.yaml:

```
# Insecure registries
  insecureRegistries: {}
  #  pocRegistry: poc.myregistry.example.com
```

And it worked as exepected. Could it be integrated?

**Comment by danielpacak** [comment]
:wave: @ecniiv Certainly we could add such value to our Helm chart!
